src/labels/CustomLabels.labels

```diff
@@ -1576,6 +1576,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>The job can only be executed when Enhanced RD is enabled in the org</shortDescription>
         <value>You must enable Enhanced Recurring Donations before you can run the Enhanced Recurring Donations Data Migration job.</value>
     </labels>
+    <labels>
```

Comment:
@sam-knox Would you mind reviewing thees two labels? Ideally an Admin will never see these in their org. They're here in case someone makes an attempt to manually execute the wrong RD jobs based on whether RD2 is enabled or not.